packages/sit-onyx/src/components/OnyxTab/OnyxTab.vue

```diff
@@ -0,0 +1,117 @@
+<script lang="ts" setup>
+import { computed, inject } from "vue";
+import { useDensity } from "../../composables/density";
+import { TABS_INJECTION_KEY } from "../OnyxTabs/types";
+import type { OnyxTabProps } from "./types";
+
+const props = defineProps<OnyxTabProps>();
+
+defineSlots<{
+  /**
+   * Tab panel / content.
+   */
+  default(): unknown;
+  /**
+   * Optional slot to override the tab content. By default, the `label` property will be displayed.
+   */
+  tab?(): unknown;
+}>();
+
+const { densityClass } = useDensity(props);
+const tabsContext = inject(TABS_INJECTION_KEY);
+
+const isActive = computed(() => tabsContext?.headless.state.isVisible.value(props.value) ?? false);
```

Comment:
We should use the same name for the state (*active* vs. *selected*) to reduce cognitive overhead:
```suggestion
const isSelected = computed(() => tabsContext?.headless.state.isSelected.value(props.value) ?? false);
```